docs/extending/data-source.md

```diff
@@ -53,13 +53,29 @@ $zipcode_query = HttpQuery::from_array( [
 ])
 ```
 
-In this case the `data_source` has a built in `get_endpoint()` method. You could also retrieve the endpoint this way:
+In this case the `data_source` has a built in `get_endpoint()` method. Other configuration options can be retireved directly. If you had a Data Source config like:
+
+```php
+$comp_data_source = HttpDataSource::from_array( [
```

Comment:
Can we just standardize on a variable name like `$example_data_source` or `$data_source`? It's not particularly complicated